### PR TITLE
Modulo encoding

### PR DESCRIPTION
**Pull request by vidavakil**
Adds Modulo/Circular Encoding for inputs of Performance_RNN.

### CODE REVIEW

**Comment by vidavakil** [comment]
PTAL.

File: magenta/models/music_vae/README.md [hunk 1]
```diff
@@ -41,7 +41,6 @@ MusicVAE models in the browser. Existing applications built with this library in
 Learn more about the API in its [repo](https://goo.gl/magenta/musicvae-js).
```
Comment (iansimon) [inline magenta/models/music_vae/README.md]: Not sure how you ended up with changes to this file but can you revert them?  Looks good other than that!

Comment (vidavakil) [reply magenta/models/music_vae/README.md]: Done.

**Comment by vidavakil** [comment]
kokoro:run

**Comment by vidavakil** [comment]
Done

**Pull request merged**